kinto/config/__init__.py

```diff
@@ -9,7 +9,8 @@ def render_template(template, destination, **kwargs):
     template = os.path.join(HERE, template)
 
     folder = os.path.dirname(destination)
-    os.makedirs(folder)
+    if os.path.exists(folder) == False:
+            os.makedirs(folder)
```

Comment:
Is there too much spaces there? (should be 4 I think)
+            os.makedirs(folder)
 Styling with Markdown is supported Write Preview

Is there too much spaces there? (should be 4 I think)
Attach files by dragging & dropping,  Sélect. fichiers selecting them, or pasting from the clipboard.
Comment Cancel